static/js/app/modal.js

```diff
@@ -0,0 +1,25 @@
+/*
+@param { String } modalId
+The DOM id of the element you'd like to turn into a modal.
+*/
+window.app.modal = function(modalId) {
```

Comment:
Now that this is being included as a template it should check to see if the variable is available before assigning into it. In this case it's not really a problem but just a good practice

```
if (!window.app.model) {
   ...
}
```